### ISSUE TITLE
SSPRKNewton prints independent of log level

### ISSUE THREAD

**Issue by AlmostSurelyRob** [erdc-cm/proteus#190]
I am trying to suppress output in one of my cases, mainly for the purpose of neat IPython notebook but also for better clarity. Ideally I would Proteus to say or do something only when the case doesn't converge. Whatever I set for my 1D Burgers example I get

```
...
SSPRKNewton it 1 norm(r) 0.00126787788509
SSPRKNewton it 1 norm(r) 0.000924199653427
SSPRKNewton it 1 norm(r) 0.00164995377224
SSPRKNewton it 1 norm(r) 0.00126649483745
SSPRKNewton it 1 norm(r) 0.000926158941493
SSPRKNewton it 1 norm(r) 0.00165586094848
SSPRKNewton it 1 norm(r) 0.00126528847383
SSPRKNewton it 1 norm(r) 0.000928881820217
SSPRKNewton it 1 norm(r) 0.00117504147603
SSPRKNewton it 1 norm(r) 0.00086784593278
SSPRKNewton it 1 norm(r) 0.000639966049084
...
```
coming out of the solver. I would like to suppress that as I will need to run a number of Proteus simulations and I don't want these solver messages to flood the logs unless they denote some serious convergence issue? It it possible with the current code?

The above occurs irrespectively of the -v -l $number setup in parun or appropriate settings in the source code.

**Comment by cekees** [comment]
These are controlled by the `Profiling.logLevel` and  `Profiling.verbose`, which is  what the `-l` and `-v` options manipulate  for  you in parun, but in a notebook or python script you have to modify them directly.  I'll have to check  the SSPRKNewton implementation to make sure  it's  using logging and  not a simple print statement.

**Comment by cekees** [comment]
Yeah, sorry.  There's a rogue print statement in there: https://github.com/erdc-cm/proteus/blob/master/proteus/NonlinearSolvers.py#L992. I will  push a patch that replaces  it with `Profiling.logEvent`.

**Comment by AlmostSurelyRob** [comment]
Got it. Yeah I saw it whilst grepping through the source code, but I didn't notice that other solvers are simply using log function.

**Comment by cekees** [comment]
closed by #192
